lib/gem_of/rake_tasks.rb

```diff
@@ -181,20 +194,25 @@ def initialize
     private
 
     # @api private
-    def diff_length
-      max_length = 150
-      target_branch = ENV["DISTELLI_RELBRANCH"] || "master"
-      diff_cmd = "git diff --numstat #{target_branch}"
-      sum_cmd  = "awk '{s+=$1} END {print s}'"
-      diff_len = `#{diff_cmd} | #{sum_cmd}`.to_i
+    def log_diff_length_and_exit
+      max_length = 500
+      diff_len = diff_length
       if diff_len < max_length
         puts "diff length (#{diff_len}) is less than #{max_length} LoC"
-        return
       else
-        puts "diff length (#{diff_len}) is more than #{max_length} LoC"
-        return diff_len
+        STDERR.puts "[E]: diff length (#{diff_len}) is more than \
+                     #{max_length} LoC"
+        exit diff_len
```

Comment:
So I guess the follow up question is does it need to actually exit?  You could move the out of the method and into the task.  That would make the spec tests cleaner.  But also, does the task need to exit, or just "return" a non-zero value?    Like if I made a task called pr_test, and I called all the tasks that test my code, if one exits the other don't run.  But if they return a non-zero, then they "can" all run and the container task could or might already sum up the returns values...  Not sure about all that though...

Reply:
good question.  the task could exit with the length, i suppose.  🤷‍♂️

Reply:
but that's actually harder to test in rspec

Reply:
I will merge as is.  The change to move the exit and enable multiple tasks to be run and have it exit at the end can be a separate feature at some future time. :)